mobify/commercecloud-ocapi-client
# Add/Update Models

A partner has run into an issue where the variation attribute values doesn't have the 'image' nor 'image_swatch' properties. This PR adds the parsing of these values. As well as a few other properties.



## Type of change


## Changes

- Added `image` and `image_swatch` properties on the VariationAttributeValue model
- Added `represented_product` and `represented_products` on the ProductSearchHit model
- Created `ProductRef` model

# How to test this PR?

You can test this PR by following the below steps:
1. Clone and `npm ci` the `mobify-platform-sdks` project
2. Clone this repo (particularly this branch)
3. Link these projects by first installing and building the `commercecloud-ocapi-client` then running `npm link`
3. In the `packages/commerce-integrations` folder of the sdks project run `npm link ../../../commercecloud-ocapi-client`
4. Once these are linked you can run the explore project `packages/explorer` and previewing [here](https://preview.mobify.com/?url=https%3A%2F%2Fsfcc.merlinspotions.com&site_folder=http%3A%2F%2Flocalhost%3A3000%2Fstatic%2Fjs%2Fbundle.js&disabled=0&domain=&scope=1)
5. Search for some products and you should now see the correct content in the models.

# Checklist: 

- [ ] My code follows the style guidelines of this project (`npm run lint`)
- [ ] New and existing unit tests pass locally with my changes (`npm test`)
